Xplat/src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -2118,6 +2118,7 @@
   "botania.tagline.flowerShrinking": "Smaller flowers with a smaller range",
   "botania.page.flowerShrinking0": "Some functional flowers can have a bit too much reach for certain use-cases. Tossing one of those into a $(l:mana/pool)$(item)Mana Pool$(0)$(/l) with an $(l:devices/mana_alchemy)$(item)Alchemy Catalyst$(0)$(/l) will (at the cost of some $(thing)Mana$(0)) shrink it, making its area of effect smaller. Every other function remains the same.",
   "botania.page.flowerShrinking1": "Like a nendoroid",
+  "botania.page.flowerShrinking2": "Petite Flower",
```

Comment:
Maybe make the heading say "Petite Flowers" instead?

Reply:
I was also very hesitant about that choice.

![image](https://user-images.githubusercontent.com/43409914/201353997-a8b675d6-ef6e-4c5a-8f8b-ca0c7b5b9d1c.png)

Here for example:
- there are multiple petals (2)
- there are multiple type of petals (16)

But it's still "Mystical Petal", so i wanted to use pre-existing naming logic.

However that logic could be changed, if the rules are clearly defined.

Reply:
I think the "Mystical Petal" name here would be better off as "Mystical Petals" too, for both the reasons you specified. I can change that myself now, since it's in a separate file and doesn't really belong in this PR.

So to be clear, yeah change it to "Petite Flowers" please.

Reply:
Sure, done.